utils/utils.jsx

```diff
@@ -1571,6 +1571,13 @@ export function isValidPassword(password, passwordConfig) {
     return {valid, error};
 }
 
+export function isEmailVerified(user) {
```

Comment:
Sounds good! I was debating on whether this would be used elsewhere but since it is only used once now it makes sense to just do the check manually where its needed. :+1: